client/js/lounge.js

```diff
@@ -372,6 +408,16 @@ $(function() {
 			.find("#chan-" + data.chan)
 			.find(".messages");
 
+
+			// remove the date-change marker we put at the top, because it may
```

Comment:
Please format the comments better - have the top one directly above the bottom, not indented